pkg/errors/errors.go

```diff
@@ -232,3 +234,24 @@ func NewUserNotAllowedError(image string, onbuild bool) error {
 		Suggestion: fmt.Sprintf("modify image %q to use a numeric user within the allowed range or build without the --allowed-uids flag", image),
 	}
 }
+
+// NewMissingGitBinaryError returns a new error that indicates that we couldn't
+// find a Git binary in the path.
+func NewMissingGitBinaryError() error {
```

Comment:
@php-coder We could use it when hasGitBinary() is callled on line 251 of git/git.go, but I think we talked about whether we should error out when the user doesn't have Git installed and decided that we would just use --copy instead.  

Are we tight about removing unused code?  Imo it's a legitimate error, and we may use it if we decide otherwise.  I'm not very opposed to just nixing it, thought.